README

```diff
@@ -41,32 +41,57 @@ The callback will only report on the status of tasks that have a name beinging w
 
 Note::
     If you haven't deployed using infrared, you can still run the tests if you create your own inventory file, containing one group of hosts called ``overcloud_nodes``.
-
+    Alternatively, you can create your own playbook that imports the tasks in ``tasks/*.yml``
 
 Adding new tests
 ----------------
 
-Adding a new test for existing deployment templates requires no changes in Jenkins.
-Adding tests for new deployment templates requires that the template is in Infrared and deployed in Jenkins.
+Adding a new test for existing deployment templates requires no changes in
+Jenkins.
+Adding tests for new deployment templates requires that the template is in
+Infrared and deployed in Jenkins.
+
+The changes required in this repo for tests is to add additional tasks under
+``tasks/test_<your_test_name>.yml``.
+
+```
+     - name: "[Test] My test name"
+       shell: |
+           my_test_command
+       register: command_output
+       failed_when: command_output.stdout == some_value
+
+```
+
+Most of the functional tests are a series of shell commands that one would run
+when verifying that particular features were deployed correctly, and can be
+adapted from any manual testing that is done.
+
 
-The changes required in this repo for tests is a new role in ``run_stf_tests.yml`` with a tag, e.g.::
+The task can then be imported in ``stf_functional_tests.yml`` like so::
 
 ```
-    - name: Collectd checks
+     - name: Collectd checks
```

Comment:
This offset I think looks weird? Extra space?

Reply:
Should be taken care of, I think it was a yank and paste issue